testsuite/manualmode/src/test/java/org/jboss/as/test/manualmode/management/cli/ReloadOpsTestCase.java

```diff
@@ -108,55 +94,13 @@ public void testWriteAttribvuteWithReload() throws Exception {
         assertThat(value, is("false"));
     }
 
-    private void reloadServer(long timeout) throws Exception {
-        executeReload();
-        waitForLiveServerToReload(timeout);
```

Comment:
There (w)are lots of similar problems all around our tests, not sure when this was initially introduces, but I agree it shouldn't be needed.